packages/front-end/components/Experiment/TabbedPage/ExperimentHeader.tsx

```diff
@@ -541,103 +541,105 @@ export default function ExperimentHeader({
         </div>
       </div>
 
-      <div
-        className={clsx(
-          "experiment-tabs bg-white px-3 border-bottom d-print-none",
-          {
-            pinned: headerPinned,
-          }
-        )}
-      >
-        <div className="container-fluid pagecontents position-relative">
-          <div className="row align-items-center header-tabs">
-            <div
-              className="col-auto pt-2 tab-wrapper"
-              id="experiment-page-tabs"
-            >
-              <TabButtons className="mb-0 pb-0">
-                <TabButton
-                  active={tab === "overview"}
-                  display={
-                    <>
-                      <FaHome /> Overview
-                    </>
-                  }
-                  anchor="overview"
-                  onClick={() => setTab("overview")}
-                  newStyle={false}
-                  activeClassName="active-tab"
-                />
-                <TabButton
-                  active={tab === "results"}
-                  display={
-                    <>
-                      <PiChartBarHorizontalFill /> Results
-                    </>
-                  }
-                  anchor="results"
-                  onClick={() => setTab("results")}
-                  newStyle={false}
-                  activeClassName="active-tab"
-                  last={false}
-                />
-                {isBandit && (
+      {experiment.status !== "draft" ? (
```

Comment:
It's possible to revert a running experiment back to a draft.  In that case, there would be results and we would want to show the tabs.  We should change this conditional to check for draft AND no results yet.

Reply:
also its possible there could be results in previous phases but not the current phase, and they're wanting to look back at previous results